### DISCUSSION TITLE
TipTap Markdown

### DISCUSSION THREAD

**Discussion by futurewebsites** [area17/twill#1448] [General]
Hi Folks does anyone know if its possible to configure the wysiwyg editor to use Tiptap but as markdown? or if there is anyway to get a Markdown editor?

**Comment by kallefrombosnia** [comment]
From their documentation, we can see that Tiptap [doesn't fully support Markdown syntax.](https://tiptap.dev/guide/output#not-an-option-markdown)
But they allow some [features like Markdown to style input](https://tiptap.dev/examples/markdown-shortcuts).
Maybe you already have seen this info, but just in case to confirm it.

I think there was a discussion about changing the current WYSIWYG editor, but I could be wrong. For the current default Quill editor, there are a few plugins on [npm](https://www.npmjs.com/package/quilljs-markdown), but for implementation, you would need to check how the current component works and how to load it.

https://github.com/area17/twill/blob/2.x/frontend/js/components/Wysiwyg.vue

After manual changes, you will need to recompile Vue components.

https://twill.io/docs/block-editor/development-workflow.html#a-bit-further-extending-twill-with-custom-components-and-custom-workflows

I'm sure @ifox and @pboivin can help you with this since they are more experienced in this field.
